gatekeeper/apitypes/bootstrap.go

```diff
@@ -0,0 +1,32 @@
+package apitypes
```

Comment:
I wasn't sure if this should go in it's own package under `gatekeeper`, or added to the `apitypes` base package. Thoughts?